plugins/linux-capture/xcompcap-main.cpp

```diff
@@ -135,7 +135,7 @@ void XCompcapMain::defaults(obs_data_t *settings)
 	obs_data_set_default_bool(settings, "exclude_alpha", false);
 }
 
-#define FIND_WINDOW_INTERVAL 0.5
```

Comment:
Is this change related to the fix?

Reply:
Oops, I accidentally pushed some debug code. It is now removed.